### DISCUSSION TITLE
Do we have pre-computed embeddings of known protein sequences available for download?

### DISCUSSION THREAD

**Discussion by johnnytam100** [facebookresearch/esm#212] [Q&A]
I am trying to prepare the esm1b embeddings for UniRef50.
Do we have such pre-computed embeddings available for download?
Thanks!

**Comment by tomsercu** [comment]
We do not have these computed or hosted anywhere at the moment. But completely doable, should be couple of 100s of GB for Uniref50 (40M seqs x 1024 embedding dimension x 4bytes per float).

**Reply by tomsercu** [reply]
In case you want to use this to search for sequences, this thread may be relevant: https://github.com/facebookresearch/esm/issues/23

**Reply by johnnytam100** [reply]
Luckily, I found this -> http://sigmat.ugr.es/~amelia/FoldEmbeddings/
I will first download the embeddings to see how the data goes! :)
